www/docs/client/links/index.md

```diff
@@ -0,0 +1,102 @@
+---
+id: index
+title: Links Overview
+sidebar_label: Overview
+slug: /links
+---
+
+Links enable you to customize the flow of data between the tRPC Client and Server. The way links are implemented in tRPC is very similar to Apollo's links. We will cover the basic concepts here, but if you need to learn about links more in-depth we recommend to check out Apollo's [links docs](https://www.apollographql.com/docs/react/api/link/introduction/).
```

Comment:
I don't like calling out to Apollo's docs here, we should maybe just copy that section into our own docs if it's good.

Reply:
I actually recreated the diagram for tRPC in Excalidraw but didn't know if it would be good to add it given that it would seem like we are just copying Apollo's docs. (It's also just an image instead of an svg that reacts when you toggle the theme like on Apollo's)

<img width="1113" alt="Screen Shot 2022-10-05 at 9 59 17 AM" src="https://user-images.githubusercontent.com/22778956/194093322-f500abe6-02f7-41b0-bfe3-eac71a53742d.png">

Reply:
That looks good! We just should just credit Apollo beneath the diagram.

Reply:
Alright, I'll add it then! Just to know, how do you manage assets in the docs? I ask because I see that you have an "assets.trpc.io" for the videos on the home page and don't have any images as far as I've seen. Do I just add an "image" directory with the diagram in "www"?

Reply:
`www/static/img` should be fine

Reply:
ah sorry, didn't see it there

Reply:
done!